packages/next/build/webpack/loaders/css-loader/src/runtime/api.js

```diff
@@ -0,0 +1,95 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+// css base code, injected by the css-loader
+// eslint-disable-next-line func-names
+module.exports = function (useSourceMap) {
+  const list = []
+
+  // return the list of modules as css string
+  list.toString = function toString() {
+    return this.map((item) => {
```

Comment:
```suggestion
    return this.map(function (item) {
```
It looks like we need to ensure we don't use any non IE11 compatible syntax/features here since this isn't transpiled

Reply:
I used the same css-loader version that Next.js already has `v4.3.0`: https://github.com/webpack-contrib/css-loader/blob/v4.3.0/src/runtime/api.js

Reply:
Ah they compile it using Babel: https://unpkg.com/css-loader@4.3.0/dist/runtime/api.js